crates/sui-framework/sources/coin.move

```diff
@@ -224,6 +224,17 @@ module sui::coin {
         )
     }
 
+    /// Split coin `self` into `n` coins with equal balances, or as close as
+    /// possible, if the balance is not evenly divisible by `n`.
```

Comment:
I think it is useful to document where the remainder ends up in the "not evenly divisible" case (I think the answer is "in `self`"?).

Reply:
Done.